core/market/src/negotiation/common.rs

```diff
@@ -431,7 +431,14 @@ impl CommonBroker {
 
         dao.terminate(&agreement_id, msg.reason, owner_type)
             .await
-            .map_err(|_e| RemoteAgreementError::InternalError(agreement_id.clone()))?;
+            .map_err(|e| {
+                log::info!(
```

Comment:
I would say it is even `warn`, because it would leave Provider and Requestor in inconsistent state